src/modules/formBuilder/components/itemEditor/conditionals/ManageEnableWhen.tsx

```diff
@@ -56,44 +53,24 @@ const ManageEnableWhen: React.FC<ManageEnableWhenProps> = ({
   return (
     <CardGroup
       onAddItem={() => {
-        if (fields.length === 0) {
-          // when adding first condition, set default enable behavior
-          onChangeEnableBehavior('ALL');
-        }
+        setValue(enableBehaviorPath, EnableBehavior.All);
```

Comment:
I think you still need this `if` condition or else its randomly setting it back to `ALL` each time you add a rule, even if you already specified you want `ANY`